munit/non-jvm/src/main/scala/munit/internal/junitinterface/CustomFingerprint.scala

```diff
@@ -4,8 +4,9 @@ import sbt.testing.SubclassFingerprint
 
 class CustomFingerprint(
     val suite: String,
-    val isModule: Boolean
+    _isModule: Boolean
```

Comment:
What's the deal with all these underscores changes?

Reply:
In scala3 you can't omit parens if the method was declared with them. It also applies to overriding by vals.
Check this example - https://gist.github.com/kpodsiad/8242ad1ff9eb959f3632dc63e3965f03. It will yield
```
[error]   Hello.scala:8:16: error overriding method name in trait Framework of type (): String;
[error]   value name of type String has incompatible type
```

Reply:
Ok, was this enforced in 3.1? Because apparently it was working fine in 3.0

Reply:
Ah no, wait, these files were never compiled with scala 3, got it